examples/threathunter/threat_intelligence/stix_taxii.py

```diff
@@ -180,8 +180,16 @@ def import_collection(self, collection):
         advance = True
         reports_limit = self.config.reports_limit
         logging.info(f"reports limit: {reports_limit}")
-        feed_helper = FeedHelper(self.config.start_date,
-                                 self.config.size_of_request_in_minutes)
+        if not self.config.start_date:
```

Comment:
Instead of hardcoding the start date , I think it would be a good idea to set it to the current date the script was ran. Let me know what you think!

Reply:
I like the idea, but it might not make sense for the purpose of this script. 

Basically, there needs to be a default date to start polling from, unless the user specifies a start_date. The start date can be set via the configuration file, or via the command line. If we used the current date as default, only data from today would be pulled down, which doesn't always make the most sense for users looking to import STIX data into their CB environment.

Reply:
@llyon-cb That's what I was thinking, but I didn't know how to phrase it properly.

Reply:
So then why don't we do it a month out from the current date? I am just not a fan of hardcoding variables. Id User A runs it today , they will be pulling data from a monthish ago but if someone runs it in 3 months they are pulling more than that. It's best to have consistent behavior.

Reply:
Maybe this just needs to be `raise Error()` such that if no date is specified the code doesn't execute?

Reply:
This is all he would have to do:

date.today() + relativedelta(months=+1)

You can make a note saying the code will default to a month from the day the script was ran if no date is given.

Reply:
I'm open to making `start_date` required, or making it a default value of now - [some time period].

If we do the default value of now - [some time period], does one month make the most sense? Why not two years? Many TAXII servers have data for many years. It feels like any choice we make will be arbitrary, and not very helpful to a user.

I'm more inclined to error out if no `start_date` is provided, mostly because it feels less arbitrary.

Reply:
My main argument is to have consistent behavior throughout the script. I will be okay with  an error being raised.

Reply:
I've updated this, now `start_date` is required and the script errors out for sites without `start_date`s

Reply:
Sounds good!